mParticle-iterable/IterableMPHelper.h

```diff
@@ -0,0 +1,36 @@
+//
+//  IterableMPHelper.h
+//  Pods
+//
+//  Created by David Truong on 4/21/17.
+//
+//
+
+#ifndef IterableMPHelper_h
+#define IterableMPHelper_h
+
+typedef void (^ITEActionBlock)(NSString *);
+
+#define ITBL_DEEPLINK_IDENTIFIER @"/a/[a-zA-Z0-9]+"
```

Comment:
Yes we only track the deeplinks if the link matches the pattern containing '/a/'